pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -74,10 +88,27 @@ let
     rev = "refs/tags/v1.14.1";
     hash = "sha256-ZVSdk6LeAiZpQrrzLxphMbc1b3rNUMpcxcXPP8s/5tE=";
   };
+
+   cutlass = fetchFromGitHub {
+    owner = "NVIDIA";
+    repo = "cutlass";
+    rev = "v3.0.0";
+    sha256 = "sha256-YPD5Sy6SvByjIcGtgeGH80TEKg2BtqJWSg46RvnJChY=";
+   };
+
+   microsoft-gsl' = microsoft-gsl.overrideAttrs (oa: {
+     patches = lib.optionals cudaSupport [
+       # nvcc doesn't recognize the "gsl" attribute namespace (microsoft/onnxruntime#13573)
+       (fetchurl {
+         url = "https://raw.githubusercontent.com/microsoft/onnxruntime/v${version}/cmake/patches/gsl/1064.patch";
+         sha256 = "sha256-0kv4z68mBuwrVe7UjTBbe0k7c1EAqm80JFaHrrPm3U0=";
+       })
+     ];
+   });
```

Comment:
Can't we just apply this patch in `microsoft-gsl` unconditionally?

Reply:
Also, the hash would change with the `version` and it'd be inconvenient to update it locally (because it's inside a list)

Reply:
Also would be nice to have at least one concrete link (with the rendered version) in a comment so it's easier to open and inspect

Reply:
also `s/sha256 =/hash =/`

Reply:
And although this is static content, still could `fetchpatch` to normalize the content prior to hashing (a hypothetical scenario: we later fetch a commit patch generated on the fly by github which happens to describe the same change, and with `fetchpatch` we'd have a hit; pretty unrealistic, but as a general rule why not)

Reply:
Wrote you on Matrix to discuss how to proceed

Reply:
Lgtm, and no mass rebuilds either